tests/integration_tests/scheduler/bin/runner

```diff
@@ -18,3 +18,7 @@ bash "${job}.script" > "${job}.stdout" 2> "${job}.stderr" &
 child_pid=$!
 wait $child_pid
 echo $? > "${job}.returncode"
+
+cat ${job}.stdout
```

Comment:
Is this something forgotten or printing stdout is really required?

Reply:
It is required for the tests for stdout/stderr. It was not in place for this "runner" because it is not working properly (at least not for testing) for the OpenPBS driver.